<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDlg</class>
 <widget class="QDialog" name="UserDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register user dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QLineEdit" name="mRefNumText"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="mAuthCodeLabel">
     <property name="text">
      <string>AuthCode</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="mAuthCodeText"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>User Information</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mRefNumLabel">
     <property name="text">
      <string>RefNum</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="mEmailText"/>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="mCloseBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="mDeleteBtn">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mRegisterBtn">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="mModifyBtn">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="mNameText">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mSSNText"/>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>SSN</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QComboBox" name="mStatusCombo"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="mStatusLabel">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
